.github/workflows/rune_test.yml

```diff
@@ -0,0 +1,147 @@
+name: Rune Test
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+on: [push, pull_request]
+
+env:
+  RUNE_VERSION: 0.5.0
```

Comment:
Can we remove the version in this file? Otherwise, we need to update this file each month.